Makefile

```diff
@@ -60,8 +60,11 @@ composer.lock: composer.json
 vendor: composer.lock
 	$(PHP_RUN) -d memory_limit=4G /usr/local/bin/composer install
 
+autoload:
+	$(PHP_RUN) -d memory_limit=4G /usr/local/bin/composer dump-autoload
+
 .PHONY: dependencies
-dependencies: vendor node_modules
+dependencies: vendor node_modules autoload
```

Comment:
This doesn't key off the actual autoload files. Would you want to have it check stamps on those?

Something like
```
autoload: vendor/autoload.php vendor/autoload_*.php
```

Thoughts?

Reply:
You're right it doesn't.

There isn't an 'autoload' file or dir at the top level, so this gets re-run every time. I'm OK with that cause it's fast and this isn't something we run a lot (just on boot)

Reply:
Sorry, edited. I meant the composer autoload files. I think autoload.php is just a proxy to those.

Reply:
I'm still OK with this running every time cause it's fast (0.16s in dev).